test/controllers/api/profiles/solutions_controller_test.rb

```diff
@@ -63,7 +63,7 @@ class API::Profiles::SolutionsControllerTest < API::BaseTestCase
     setup_user
 
     profile_user = create(:user_profile).user
-    5.times { create :practice_solution, :published, user: profile_user }
+    5.times { |i| create :practice_solution, :published, user: profile_user, num_stars: i }
```

Comment:
This test was failing on my machine, and it had to do with the sorting. All solutions had the same number of stars (which is the sorted field), but somehow the returned and the expected value were different. This assigns them a unique number of stars.